docs/set_esxi_subnets.md

```diff
@@ -1,14 +1,14 @@
-# set_esx_subnets
+# set_esxi_subnets
 
 Sets the subnets that should be used to reach the ESXi hosts.
 ```py
-def set_esx_subnets(esx_subnets=None, timeout=15)
+def set_esxi_subnets(esx_subnets=None, timeout=15)
 ```
 
 ## Keyword Arguments
 | Name        | Type | Description                                                                 | Choices | Default |
 |-------------|------|-----------------------------------------------------------------------------|---------|---------|
-| esx_subnets | str  | Preferred subnets used to reach the ESX hosts. The format should be the comma separated list without any spaces, for example, 192.168.2.10/24,10.255.0.2/16.  |         |         |
+| esx_subnets | list | Preferred subnets used to reach the ESX hosts. The format should be a list, for example ['192.168.2.10/24','10.255.0.2/16'].  |         |         |
```

Comment:
`The format should be a list, for example ['192.168.2.10/24','10.255.0.2/16']`

nit: We already define it as a list so we can just provide the example here.

Reply:
👍